.travis.yml

```diff
@@ -63,6 +63,20 @@ jobs:
       script:
         - vendor/bin/phpcs
 
+    - stage: Ensure API BC
+      php: 7.3
+      env: DEPS="high"
+      script:
+        - composer require --dev roave/backward-compatibility-check:^4.0.0
```

Comment:
Won't work due to conflicting deps: will try making a new dir and installing it there

```
./composer.json has been updated

Loading composer repositories with package information

Updating dependencies (including require-dev)

Your requirements could not be resolved to an installable set of packages.

  Problem 1

    - Installation request for roave/backward-compatibility-check ^4.0.0 -> satisfiable by roave/backward-compatibility-check[4.0.0].

    - Can only install one of: phpdocumentor/type-resolver[1.0.1, 0.4.0].

    - Can only install one of: phpdocumentor/type-resolver[0.4.0, 1.0.1].

    - Can only install one of: phpdocumentor/type-resolver[0.4.0, 1.0.1].

    - Conclusion: install phpdocumentor/type-resolver 0.4.0

    - Installation request for phpdocumentor/type-resolver (locked at 1.0.1) -> satisfiable by phpdocumentor/type-resolver[1.0.1].
```

Reply:
Yeah, this should just be on circleci I think